### DISCUSSION TITLE
Compressed plots - proof drops

### DISCUSSION THREAD

**Discussion by vxl18** [Chia-Network/chia-blockchain#17008] [Support]
Hello,
I have migrated my farm fully to BladeBit compressed plots (C5 compression level).

Since I started the migration process, I have noticed new INFO messages in debug.log file:
_harvester chia.harvester.harvester: INFO     Proof dropped due to line point compression for <path_to_plot>._

It occurs few times every day (5-15 times a day).

Searching web did not give any answers about these messages.

What does it mean? Should I remove these plots and create new ones?

Thank you!

**Comment by BrandtH22** [comment]
Hey @vxl18 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

I would run the `chia plots check -g <path_to_plot>` command on the plot(s) that are yielding that error (https://docs.chia.net/cli#plots-check) to check the quality of the plot (note that running this command while farming and/or plotting can yield incorrectly poor results, generally best to run this command when nothing else is running on the machine).

If the plot is bad or of poor quality you will likely want to remove it and create a new plot in its place.

**Reply by vxl18** [reply]
**BrandtH22**, thank you for response!
I will try it in a while.

**Reply by BrandtH22** [reply]
Hey @vxl18 , were you able to run the chia plots check command and determine whether this was a bad plot or not?

**Reply by vxl18** [reply]
Hi, @BrandtH22 , 
Yes, I ran the test and "chia plots check" showed proof drops. I replaced this plot with the new one.
I will continue to watch log for these drops and replace "invalid" plots.

**Reply by BrandtH22** [reply]
Hey @vxl18 very glad to hear you were able to get that plot replaces.
I am going to close this ticket for the time being but do not hesitate to reopen it or join us on discord if any additional questions arise :+1:
